README.md

```diff
@@ -248,6 +248,24 @@ connection.connect();
 ...
 ```
 
+#### TLS PSK with proxy
+
+1. Config mqtt broker to load tls psk config.
+```java
+MQTTServerConfiguration mqtt = new MQTTServerConfiguration();
+mqtt.setMqttProxyEnable(true);
+mqtt.setMqttProxyTlsPskEnabled(true);
+mqtt.setTlsPskIdentityHint("alpha");
+// psk identity is semicolon list of identities with identity:serect format
+mqtt.setTlsPskIdentity("mqtt:mqtt123");
```

Comment:
We should change to config params in the config file, not the Java API.

Reply:
Done.